lib/internal/cluster/primary.js

```diff
@@ -65,7 +67,8 @@ cluster.setupPrimary = function(options) {
     execArgv: process.execArgv,
     silent: false,
     ...cluster.settings,
-    ...options
+    ...options,
+    ...{ schedulingPolicy: processSchedulingPolicy(options) }
```

Comment:
Why the object literal here instead of setting inline?

Reply:
```suggestion
    schedulingPolicy: processSchedulingPolicy(options),
```